README.md

```diff
@@ -28,14 +28,25 @@ Usage:
     -c         : list CPU temperatures (Celsius)
     -g         : list GPU temperatures (Celsius)
     -h         : help
+    -i         : set interval in milliseconds (e.g. -i25, valid range is 20-1000, default: 1000)
     -l         : list all keys and values
     -f         : fail-soft mode to store recent valid sensor value
     -v         : version
-    -n         : tries to query the temperature sensors for n times (e.g. -n3) (1 second interval) until a valid value is returned
+    -n         : tries to query the temperature sensors for n times (e.g. -n3) until a valid value is returned
 
 $ smctemp -c
 64.2
 
 $ smctemp -g
 36.2
 ```
+
+## Note for M2 Mac Users
+On M2 Macs, sensor values may be unstable as described in the following issue:
+- https://github.com/narugit/smctemp/pull/14
+- https://github.com/narugit/smctemp/issues/32
+
+For M2 Macs, using the `-n`, `-i`, and `-f` options can help obtain more stable sensor values.
+Try tuning these options to get better results.
+
+The recommended settings are `-i25 -n40 -f` (See also https://github.com/narugit/smctemp/issues/32#issuecomment-2287304793).
```

Comment:
Hi @kyleyannelli 
I'd like to add recommended settings for M2 user, so could you provide your recommended settings with `-f`?

Reply:
Hi @narugit
After messing around a bit I'm going to recommend `-i25 -n180 -f`. The lower retries just don't hit even when the CPU is under load. I think 25ms and 180 maximum retries work well enough to get a result while the max wait time is under 5 seconds.

Reply:
@kyleyannelli 
Thank you for checking! I've updated the recommended settings.